share_extension/components/image_file.tsx

```diff
@@ -4,16 +4,24 @@
 import React from 'react';
 import {StyleSheet, View} from 'react-native';
 import FastImage from 'react-native-fast-image';
+import validator from 'validator';
 
 interface ImageFileProps {
     uri: string;
 }
 
 const ImageFile = ({uri}: ImageFileProps) => {
+    const config: validator.IsURLOptions = {
+        protocols: ['file', 'http', 'https'],
```

Comment:
content ?

Reply:
I'll add / check for `content://` and `data://` types 👍